### PR TITLE
Bugfix align empty pair tags

### PR DESCRIPTION
**Pull request by mariagardbraga**
o uatp esta dependente deste branch ser merged com master.

### CODE REVIEW

File: unbabel_text_utils/inline_to_annotations/inline_to_annotations.py [hunk 1]
```diff
@@ -111,14 +111,13 @@ def annotation_to_inline(text,
...
         intervals_tagged_text = get_list_to_insert_tags(text, markup)
-
     else:
         intervals_tagged_text = [[text, False]]
     tagged_intervals = []
     start = 0
     end = 0
     spaces_added = 0
-
+    print "intervals tagged text", intervals_tagged_text
```
Comment (vascop) [inline unbabel_text_utils/inline_to_annotations/inline_to_annotations.py]: Let's get rid of debug print statements

File: unbabel_text_utils/tags_align/align_wa/tags_region.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
+# -*- coding: UTF-8 -*-
+from HTMLParser import HTMLParser
+from string import punctuation
```
Comment (vascop) [inline unbabel_text_utils/tags_align/align_wa/tags_region.py]: This import is unused

File: unbabel_text_utils/tags_align/align_wa/tags_region.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                for str_pos, list_tags in tags.iteritems():
+                    for tag in list_tags:
+                        if region[1] == tag['tid']:
+                            tag['tag_type'] = 'open_empty'
+                            break
+            continue
+        list_last_word_tokens = list_duplicates_of(tokens, region_tok[-1])
+        aux_last_word = []
+        for i in list_last_word_tokens:
+            if (i >= open_word_pos) and (i - open_word_pos <= len(region_tok)+1):
```
Comment (vascop) [inline unbabel_text_utils/tags_align/align_wa/tags_region.py]: pep8

**Comment by vascop** [comment]
I've fixed the pep8 things. We should talk about the tests, commenting them isn't ever the solution.

**Comment by mariagardbraga** [comment]
@vascop Tests are set for the new format and it seems they passed :)

**Pull request closed without merging**